BookingsApi/BookingsApi.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -54,6 +55,27 @@ public TestDataManager(DbContextOptions<BookingsDbContext> dbContextOptions, str
             return SeedVideoHearing(null, addSuitabilityAnswer, status, isMultiDayFirstHearing:isMultiDayFirstHearing);
         }
 
+        public async Task<JusticeUser> SeedJusticeUser(string userName, string firstName, string lastName, bool isTeamLead = false)
+        {
+            await using var db = new BookingsDbContext(_dbContextOptions);
+            var justiceUser = db.JusticeUsers.Add(new JusticeUser
+            {
+                ContactEmail = userName,
+                Username = userName,
+                UserRoleId = isTeamLead ? 9 : 2,
```

Comment:
Might be worth creating an enum for the userRoleID or a public const for the team lead user id, which can be accessed in all the other tests